src/index.js

```diff
@@ -1,3 +1,5 @@
+import 'babel-polyfill' // Array.prototype.values, etc.
```

Comment:
this library has to be minimal, therefor no "babel-polyfill" or usage of native ES2016 features like .values or .includes